fixed README.md

```diff
@@ -31,20 +31,22 @@ This will automatically download and install the packages listed in the `require
 
 <h2 id="3.2. .env File">3.2. `.env` File</h2>
 This bot relies on a `.env` file being present, a template can be seen below, **both variables are required**.
-```
 
+```
 DISCORD_TOKEN=""
 CHANNEL_ID=
-
 ```
+
 <h3 id="3.2.1. DISCORD_TOKEN">3.2.1. `DISCORD_TOKEN`</h3>
 This is provided by Discord and can be obtained by:
-1. Going to https://discord.com/developers/applications`.
-2. Clicking on the bot.  If you have not yet created a bot, then click `New Application` in the top right.
-3. Navigate to the `Bot` menu on the left-hand side
-4. Click `Copy` under `Click to Reveal Token` and paste into the `.env` file.  If you haven't created a bot, then press `Add Bot`, then `Yes, do it!`.
-
-<h3>3.2.2. `CHANNEL_ID`</h2>
+ <ol>
+  <li>Going to https://discord.com/developers/applications`.</li>
+  <li>Clicking on the bot.  If you have not yet created a bot, then click `New Application` in the top right.</li>
+  <li>Navigate to the `Bot` menu on the left-hand side.</li>
+  <li>Click `Copy` under `Click to Reveal Token` and paste into the `.env` file.  If you haven't created a bot, then press `Add Bot`, then `Yes, do it!`.</li>
+</ol>
+
+<h3 id="3.2.2. CHANNEL_ID">3.2.2. `CHANNEL_ID`</h2>
```

Comment:
I'm using HTML in the README file for the contents page so you can jump to the section headers.
Probably isn't needed but I threw it in for uh... reasons... definately not because I was bored...